collectors/python.d.plugin/anomalies/anomalies.chart.py

```diff
@@ -52,8 +52,11 @@ def __init__(self, configuration=None, name=None):
         self.model_params_init()
         self.models_init()
 
-    @staticmethod
-    def check():
+    def check(self):
+        _ = get_allmetrics_async(
```

Comment:
does `get_allmetrics_async` throw an exception?

if so we are good, if not need to return `bool(returned_data)`

Reply:
get_allmetrics_async() does not handle exceptions internally so will just fail if anything goes wrong. Which I believe is what we want here.